.idea/gradle.xml

```diff
@@ -14,7 +14,6 @@
           </set>
         </option>
         <option name="resolveModulePerSourceSet" value="false" />
-        <option name="useQualifiedModuleNames" value="true" />
```

Comment:
I believe the changes within the idea folder should be merged, or else it will cause lots of merge conflicts.

Add this line back, and push the the changes so that no changes are present within the idea folder.

Reply:
Ok, I have added it